src/CellData/CellFields.jl

```diff
@@ -210,6 +209,169 @@ DomainStyle(::Type{GenericCellField{DS}}) where DS = DS()
 
 # Evaluation of CellFields
 
+# This code lives here (instead of in the CellData module) because we
```

Comment:
This comment does not apply anymore, right?

Reply:
I deleted the comment from `CellFields.jl`